core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -3820,6 +3820,29 @@ public void checkOrderBy(final boolean desc,
             "deptno=20; empid=200; hire_date=2014-06-12; R=1");
   }
 
+  @Test public void testNestedWin() {
```

Comment:
Should this test be added to .iq file?

Reply:
Do you mean that this test should be added to .iq file instead? May I ask what kind of test case should be put in JdbcTest.java and what kind of test case should be put in .iq file ?

Reply:
Frankly speaking I would add the test to a separate (newly created) class like `jdbc.window.WindowFunctionsTest.kt`

JdbcTest is huge, and it is better to refrain adding new tests there.
It would be better to create a new class that is focused on Window functions.

@ritesh-kapoor , @lxian , WDYT?